tutorial.md

```diff
@@ -64,5 +64,5 @@ Note: `sbom-scorecard` will guess the type if no type is specified.
 To run `sbom-scorecard` and specify the output format as JSON, run:
 
 ```bash
-go run cmd/sbom-scorecard/main.go score --outputFormat json examples/dropwizard.cyclonedx.json
```

Comment:
I am only able to run sbom-scorecard tool using this `go run` command, even after running the `go install` command above.

```
rose@rose-vm:~/ternenv/sbom-scorecard$ go install cmd/sbom-scorecard/main.go
# bin directory properly created
rose@rose-vm:~/ternenv/sbom-scorecard$ ls ~/go/
bin/ pkg/

# try to run using sbom-scorecard command doesn't work
rose@rose-vm:~/ternenv/sbom-scorecard$ sbom-scorecard --help
sbom-scorecard: command not found

# using go run does
rose@rose-vm:~/ternenv/sbom-scorecard$ go run cmd/sbom-scorecard/main.go --help
sbom-scorecard evaluates the quality of the SBOM

Usage:
  sbom-scorecard [command]

Available Commands:
  completion  Generate the autocompletion script for the specified shell
  help        Help about any command
  score       score evaluates the SBOM being passed and outputs based on the composition and completion

Flags:
  -h, --help   help for sbom-scorecard

```

Reply:
Interesting, if I call sbom-scorecard with `main` instead of `sbom-scorecard`, it works....?

```
rose@rose-vm:~/ternenv/sbom-scorecard$ go install cmd/sbom-scorecard/main.go
rose@rose-vm:~/ternenv/sbom-scorecard$ sbom-scorecard
sbom-scorecard: command not found
rose@rose-vm:~/ternenv/sbom-scorecard$ main
sbom-scorecard evaluates the quality of the SBOM

Usage:
  sbom-scorecard [command]

Available Commands:
  completion  Generate the autocompletion script for the specified shell
  help        Help about any command
  score       score evaluates the SBOM being passed and outputs based on the composition and completion

Flags:
  -h, --help   help for sbom-scorecard

Use "sbom-scorecard [command] --help" for more information about a command.
```

Reply:
Good find. I'll fix up the docs and rearrange things so this works correctly.

Reply:
Tracked here: https://github.com/eBay/sbom-scorecard/issues/21